dcos_installer/action_lib.py

```diff
@@ -360,8 +360,15 @@ def _add_prereqs_script(chain):
   exit 0
 fi
 
-if ([ x$dist != 'xrhel' ] && [ x$dist != 'xcentos' ]); then
-  echo "$dist is not supported. Only RHEL and CentOS are supported" >&2
+if ([ x$dist != 'xrhel' ] && [ x$dist != 'xcentos' ] && [ x$dist != 'xol' ]); then
+  echo "$dist is not supported. Only RHEL, CentOS and Oracle Enterprise Linux are supported" >&2
```

Comment:
I usually use "reported to work" vs "tested regularly". But I'm not sure either are great in the context of validation. 

That said, I'm not even sure we should have hard validation like this in the open source project. 

Tho maybe we can hedge and use the product maturity labels. That would make OEL compatibility experimental, while coreos, centos, rhel compatibility is stable.

The key is not using "support" in the name of the feature. It's "OEL compatibility" not "OEL support". 

Customers are supported. 
Features are experimental, preview, or stable.